### PR TITLE
[ CCXDEV-10824] Adding glitchtip to config

### PR DESCRIPTION
**Pull request by Jakub007d**
# Description

Adding GlitchTip (Sentry) option to configuration and test error log message to test it in stage after gitlab MR is merged.

Fixes # [CCXDEV-10824](https://issues.redhat.com/browse/CCXDEV-10824)

## Type of change

Please delete options that are not relevant.

- Configuration update

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: content-service.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ var (
 func startService() ExitCode {
 	serverCfg := conf.GetServerConfiguration()
 	groupsCfg := conf.GetGroupsConfiguration()
-
+	log.Error().Msg("Test Error for glitchtip")
```
Comment (juandspy) [inline content-service.go:85]: Please make sure to remove this or there will be a new error every time the service starts :)

Comment (juandspy) [reply content-service.go:85]: I mean after it's checked

**Review by Bee-lee — APPROVED** [review]

**Pull request merged**